app/services/users.js

```diff
@@ -0,0 +1,29 @@
+const models = require('../models/index');
+const logger = require('../logger');
+const passwordHelper = require('../helpers/passwords');
+const errors = require('../errors');
+
+exports.signUp = ({ email, password, first_name, last_name }) =>
+  models.user
+    .findOne({ where: { email } })
+    .then(user => {
+      if (user) {
+        throw errors.emailExistsError();
+      } else {
+        return models.user
```

Comment:
You don't need this else
```suggestion
      if (user) {
        throw errors.emailExistsError();
      }
        return models.user
```